baremaps-server/src/main/java/com/baremaps/server/Mappers.java

```diff
@@ -0,0 +1,21 @@
+package com.baremaps.server;
+
+import com.baremaps.config.tileset.Tileset;
+import com.baremaps.tile.postgres.PostgresQuery;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Mappers {
+
+  private Mappers() {
+
+  }
+
+  public static List<PostgresQuery> map(Tileset tileset) {
+    return tileset.getLayers().stream()
+        .flatMap(layer -> layer.getQueries().stream()
+            .map(query -> new PostgresQuery(layer.getId(), query.getMinZoom(), query.getMaxZoom(), query.getSql())))
```

Comment:
At some point this will have to become `getMaxzoom()` and `getMinZoom()` to align with changes in #319

Reply:
Indeed, probably some conflicts in perspective.